neo/app/views/index.scala.html

```diff
@@ -10,12 +10,13 @@
     </div>
 
     <div id='how-to'>
-      <a href="https://www.youtube.com/watch?v=XwlRm0jcsU4" target="_blank" class="btn btn-info">How to use this demo</a>
+      <a href="https://www.youtube.com/watch?v=XwlRm0jcsU4" target="_blank" class="btn btn-info btn-sm">How to use this demo</a>
     </div>
-
+      <search-bar></search-bar>
   </div>
+    <div class ="titleofweb">
```

Comment:
It was the class which could contain icon search bar and title in a single row.Now I removed it.